doc/classes/CPUParticles.xml

```diff
@@ -28,188 +28,188 @@
+		<member name="initial_velocity" type="float" setter="set_param" getter="get_param" default="0.0">
 			Initial velocity magnitude for each particle. Direction comes from [member spread] and the node's orientation.
 		</member>
-		<member name="initial_velocity_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+		<member name="initial_velocity_random" type="float" setter="set_param_randomness" getter="get_param_randomness" default="0.0">
 			Initial velocity randomness ratio. Default value: [code]0[/code].
 		</member>
-		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
+		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime" default="1.0">
 			Amount of time each particle will exist. Default value: [code]1[/code].
 		</member>
-		<member name="linear_accel" type="float" setter="set_param" getter="get_param">
+		<member name="linear_accel" type="float" setter="set_param" getter="get_param" default="0.0">
 			Linear acceleration applied to each particle in the direction of motion.
 		</member>
-		<member name="linear_accel_curve" type="Curve" setter="set_param_curve" getter="get_param_curve">
+		<member name="linear_accel_curve" type="Curve" setter="set_param_curve" getter="get_param_curve" default="null">
 			Each particle's linear acceleration will vary along this [Curve].
 		</member>
-		<member name="linear_accel_random" type="float" setter="set_param_randomness" getter="get_param_randomness">
+		<member name="linear_accel_random" type="float" setter="set_param_randomness" getter="get_param_randomness" default="0.0">
 			Linear acceleration randomness ratio. Default value: [code]0[/code].
 		</member>
-		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates">
+		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates" default="true">
 			If [code]true[/code], particles use the parent node's coordinate space. If [code]false[/code], they use global coordinates. Default value: [code]true[/code].
 		</member>
-		<member name="mesh" type="Mesh" setter="set_mesh" getter="get_mesh">
+		<member name="mesh" type="Mesh" setter="set_mesh" getter="get_mesh" default="null">
 			The [Mesh] used for each particle. If [code]null[/code], particles will be spheres.
 		</member>
-		<member name="one_shot" type="bool" setter="set_one_shot" getter="get_one_shot">
+		<member name="one_shot" type="bool" setter="set_one_shot" getter="get_one_shot" default="false">
 			If [code]true[/code], only one emission cycle occurs. If set [code]true[/code] during a cycle, emission will stop at the cycle's end. Default value: [code]false[/code].
 		</member>
-		<member name="orbit_velocity" type="float" setter="set_param" getter="get_param">
+		<member name="orbit_velocity" type="float" setter="set_param" getter="get_param" default="null">
```

Comment:
This one and `orbit_velocity_random` below should not be `null` but `0`. This probably happens because those parameters are only used when `disable_z` is true, which is not the default for (3D) CPUParticles.